### ISSUE TITLE
question about code

### ISSUE THREAD

**Issue by YananGu** [yaoyao-liu/CL-DETR#2]
Hi， I applied and ran the code successfully, but the result is not quite right, the loss does not drop, can you help me to see where is the problem. Thanks!
  
I ran the code by bash run.sh
Here is the log:
https://drive.google.com/file/d/1FWhq-eu_RcWYFtIFQlu8e05Jc9saOQKE/view?usp=sharing

**Comment by yaoyao-liu** [comment]
Hi @YananGu 

Thank you so much for your interest in our work!

The code will run an evaluation (e.g., printing out the AP) after each phase. However, I cannot see that in the log. Maybe the log file is incomplete. Could you please check that? Thanks!

Best regards,

Yaoyao

**Comment by YananGu** [comment]
yeah, I only ran the code for 15 epochs,  maybe It has not yet reached the evaluation stage. Then I found the loss does not drop,  so is the loss normal?

**Comment by misaka-mio** [comment]
The learning rate maybe has something wrong. If you change it to 5*e-5, the loss can drop. But the final accuracy is much lower than the paper report (about 10%).

**Comment by YananGu** [comment]
Hi @yaoyao-liu, this is the total log：
https://drive.google.com/file/d/16F_JW5HultoI8l18mswygoI7tDMn8RBz/view?usp=drive_link

Thanks @misaka-mio, I'll try it.

**Comment by misaka-mio** [comment]
Hi, did you get the results? @YananGu Can you share the results if you run better results? I want to know if I do something wrong in the experiment.  Thx.

**Comment by yaoyao-liu** [comment]
@YananGu @misaka-mio 

Thanks for your interest in our work!

The first phase is exactly the same as the original code of [DETR](https://github.com/facebookresearch/detr). The continual learning method is applied from the second phase. Therefore, you may try to reproduce the phase 1 results. All the hyperparameters follow the original DETR.

I will try to clean up and release the final version soon. If you have any further questions, please do not hesitate to contact me. 

Best regards,

Yaoyao

**Comment by aooating** [comment]
> @YananGu @misaka-mio
> 
> Thanks for your interest in our work!
> 
> The first phase is exactly the same as the original code of [DETR](https://github.com/facebookresearch/detr). The continual learning method is applied from the second phase. Therefore, you may try to reproduce the phase 1 results. All the hyperparameters follow the original DETR.
> 
> I will try to clean up and release the final version soon. If you have any further questions, please do not hesitate to contact me.
> 
> Best regards,
> 
> Yaoyao

What about non-traditional IOD data splits? Could you provide it?

**Comment by yaoyao-liu** [comment]
Hello @aooating 

May I know if you are referring to the continual learning splits used in this paper? If so, you may obtain the data splits by running the code. They will be generated automatically from the MS COCO dataset.

If you have any further questions, please feel free to leave a message here or send me an email: <yyliu@cs.jhu.edu>

Best regards,

Yaoyao
